docs/site/downloads/cldr-46.md

```diff
@@ -288,35 +288,39 @@ These matched the ordering of two legacy character encodings. [CLDR-16062][]
 1. [CLDR-17610][] Updated zip to include LICENSE file following file renaming
 
 ## 46.1 Changes
-Version 46.1 is a dot release.
+Version 46.1 is a dot release. The following summarizes the changes. For a full listing, see [Δ46.1](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246.1%22+ORDER+BY+priority+DESC)
 
 ### Message Format 2.0
-The major change is to bring the [Message Format 2.0 specification in Part 9]((https://www.unicode.org/reports/tr35/tr35-74/tr35.html#message-format-modifications)) to Final Candidate status.
-
-Addition standard functions and options are available:
-
- - Included number, date/time, and string functions
- - Each function and option is REQUIRED or RECOMMENDED. It may also be **Proposed** for a future release.
- - Added :u options to the :u namespace
- - Added options calendar, numberingSystem, and hour12 options to date/time functions.
- - Added timezone as a Proposed REQUIRED option for date/time functions.
- - Added :unit as a Proposed RECOMMENDED function.
-
-There are 3 tech preview implementations available
+The major change is to bring the [Message Format 2.0 specification in Part 9]((https://www.unicode.org/reports/tr35/tr35-74/tr35.html#message-format-modifications)) to Final Candidate status. The following summarizes changes from v46:
```

Comment:
I was thinking maybe instead of saying 'major change', the 'main focus of the release is....'